android/app/src/main/java/com/zulipmobile/notifications/FcmListenerService.kt

```diff
@@ -0,0 +1,22 @@
+package com.zulipmobile.notifications
+
+import com.google.firebase.messaging.FirebaseMessagingService
+import com.google.firebase.messaging.RemoteMessage
+import com.zulipmobile.MainApplication
+import com.facebook.react.ReactApplication
```

Comment:
nit: sort imports

(Android Studio can do this for you automatically: "Optimize Imports" from the actions menu, or for me Ctrl+Alt+O.)